web/Cargo.toml

```diff
@@ -32,7 +32,7 @@ profiling = []
 
 [dependencies]
 console_error_panic_hook = { version = "0.1.7", optional = true }
-generational-arena = "0.2.9"
+slotmap = "1.0.7"
```

Comment:
Can you add this to the workspace dependencies?

Reply:
Good idea!

Reply:
Done.